lib/ansible/modules/cloud/amazon/aws_step_functions_state_machine.py

```diff
@@ -0,0 +1,232 @@
+#!/usr/bin/python
+# Copyright (c) 2019, Tom De Keyser (@tdekeyser)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: aws_step_functions_state_machine
+
+short_description: Manage AWS Step Functions state machines
+
+version_added: "2.9"
```

Comment:
Apologies @tdekeyser, we've passed freeze for 2.9. Can you please bump this to 2.10? I'm happy to merge now that this has additional reviews.

Reply:
Hi there @jillr, no problem; I've bumped to version 2.10.